integration/system/utils_test.go

```diff
@@ -0,0 +1,18 @@
+package system
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/moby/moby/api/types/common"
+	"gotest.tools/v3/assert"
+)
+
+// getErrorMessage returns the error message from an error API response
+func getErrorMessage(t *testing.T, body []byte) string {
```

Comment:
Maybe we should consider adding a small function to the `request` test-utility  to help with decoding JSON responses (check status code, content-type, then unmarshal into a variable that's passed. 🤔 


Doesn't have to be for this PR, but I could have a quick peek